src/metabase/upload.clj

```diff
@@ -136,6 +164,15 @@
         ", " #"\d[\d \u00A0]*"
         ".’" #"\d[\d’]*"))))
 
+(defn- float-or-int-regex [number-separators]
+  (with-parens
+   (with-currency
+    (case number-separators
+      ("." ".,") #"\d[\d,]*\.0+"
+      ",." #"\d[\d.]*\,[0]+"
```

Comment:
I don't think we need the square braces around zero, I just cargo cult-ed them from the float regex.

Reply:
Agreed, we should remove them